Userland/Libraries/LibSemVer/Semver.h

```diff
@@ -64,14 +64,13 @@ enum class CompareType {
  */
 class SemVer {
     char const m_mmp_sep;
+    u64 m_major;
+    u64 m_minor;
+    u64 m_patch;
+    String m_prerelease;
+    String m_buildmetadata;
 
 public:
-    u64 const m_major;
-    u64 const m_minor;
-    u64 const m_patch;
-    String const m_prerelease;
-    String const m_buildmetadata;
-
```

Comment:
Should be squashed into the previous commit.

Reply:
Yeah, I will squash everything to just one commit.